### PR TITLE
Using state data to make totals

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Addresses #164

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
Need to round...thinking about the best place for that

**Comment by ldecicco-USGS** [comment]
OH! and the bar colors aren't changing correctly...Totals (blue), PS (red), and Irr (green) work, but industrial and thermo don't.

**Comment by jordansread** [comment]
round right before it is written to json?

**Comment by ldecicco-USGS** [comment]
Just making sure there aren't other calculations being done after...I think you are right.

**Comment by jordansread** [comment]
@ldecicco-USGS the colors won't change if the function doesn't find the field in the json, so maybe those categories are missing? It writes to cache/{filename}.json if you want to take a look

File: layout/js/map.js [hunk 1]
```diff
@@ -98,23 +98,40 @@ var animate_bars = function(data) {
...
-      "transform-origin": "100% 100%",
-      "transition": "all " + transitionTime + " ease-in-out"
-    };
+
     var bar = $("#bar-" + myYear);
+    var style = undefined;
+
     if (bar !== undefined) {
+      if (isNaN(scale)){ 
+        style = {
```
Comment (jordansread) [inline layout/js/map.js:110]: I guess we could have just said `isNaN(scale) then scale = 0` and this would work. Didn't see that the first time.

**Pull request merged**